src/Raven.Server/Documents/Handlers/BatchHandler.cs

```diff
@@ -516,6 +521,45 @@ public struct AttachmentStream
                 public string Hash;
                 public Stream Stream;
             }
+
+            DynamicJsonValue TransactionOperationsMerger.IRecordableCommand.Serialize()
```

Comment:
Don't need this, it isn't an actual recordable command, it is just a container

Reply:
In summary, it was agreed to treat the command as one unit and not as a collection